app/models/miq_expression.rb

```diff
@@ -1775,15 +1775,9 @@ def self.atom_error(field, operator, value)
                                                                                     :error_message => err.message}
       end
       return false
-    when :ruby
-      return _("Ruby Script must not be blank") if value.blank?
-      return false
```

Comment:
Note, we're returning false here and the change above to return early no longer returns false.  Is that right?